docs/_static/nuts-auth.yaml

```diff
@@ -116,6 +116,70 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Contract"
+  /auth/accesstoken:
+    post:
+      operationId: createAccessToken
```

Comment:
Some explanation on how this API should be exposed to the world. I believe a reverse-proxy with ssl termination is a must? Same goes for the other API's

Reply:
I added info about the required header.
Explaining client certs and so in this phase is more something we can add to the docs and vendors have to agree on together.

Reply:
I think there's some work to do separating public APIs (e.g., which other vendors communicate with) and private APIs (administration purposes). That should at least be documented (maybe in a diagram), and we might want to take some additional mitigating measures to minimize the risk of accidentally exposing administrative API's (e.g. exposing them on a different port or on another HTTP base path).

This is not specific to this PR or engine though but applicable to all of Nuts.